docs/config/index.rst

```diff
@@ -293,8 +297,14 @@ following content would be added to the config file template:
     ## Description of another_setting
     # another_setting = foo
 
+    ## Description of some_list
```

Comment:
@orionlee , is this a good compromise?

Reply:
Having concrete examples definitely helps, irrespective of links to configobj doc.

For links to configobj doc:
- I presume astropy uses the v5 (the `configobj.readthedocs.io` version) based on [commit comments](https://github.com/astropy/astropy/commits/main/astropy/extern/configobj/__init__.py). 
- If it's determined that voidspace.org version of the doc is the one that should be referenced, here is the correspond link for config file format:
http://www.voidspace.org.uk/python/configobj.html#the-config-file-format

Reply:
Ah, #3165 was before my time. But my concern is the `readthedocs` content does not cover other topics that the `voidspace` one does. @saimn , what do you think?

Reply:
The latest release on PyPI is from 2014. There is some activity on Github so maybe the docs on RTD is documenting some unreleased features but also probably more accurate. Not sure what is best.

Reply:
Since there is no consensus on the doc links, can we defer it in favor of getting the actual bug fix in first?

Reply:
Let's get the bug fix merged. The example in the revised doc also helped greatly. Fixing the doc links would be a plus but not a showstopper.

Reply:
Thanks for your understanding. I have opened a follow-up issue at #12109.